package.json

```diff
@@ -16,7 +16,7 @@
   "devDependencies": {
     "@vue/test-utils": "^1.0.0-beta.30",
     "accounting": "^0.4.1",
-    "axios": ">=0.19.0",
```

Comment:
Why was the version specifier change to `>=`? This would include future breaking changes as well. The `^` will ensure a compatible version is installed, and I think it can be left the way it is.

Reply:
I didn't do it specifically. I did npm update and that was the change.

Reply:
Interesting, I didn't think it would change it so drastically. In either case, the lockfile will keep the version consistent so it's not a problem.